### PR TITLE
feat(ENG-1850): update cluster advanced settings with static_ip_mode …

### PR DESCRIPTION
**Pull request by pggb25**
…and allowed_public_access_cidrs

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -13021,6 +13021,22 @@ components:
         storageclass.fast_ssd:
           type: string
           description: storage class name to use to provision pvc
+        qovery.static_ip_mode:
+          type: boolean
+          x-stoplight:
+            id: i7u9vjd0o89g8
+          description: To limit public access from the internet to your Kubernetes cluster endpoint.
```
Comment (acarranoqovery) [inline openapi.yaml:13028]: can you add that whitelisted CIDR is defined in `k8s.api.allowed_public_access_cidrs` ?

File: openapi.yaml [hunk 2]
```diff
@@ -13021,6 +13021,22 @@ components:
...
+        k8s.api.allowed_public_access_cidrs:
+          type: array
+          x-stoplight:
+            id: w7d7zyd1de9n6
+          description: |-
+            Set custom sources to public access endpoint
+            Use CIDR notation to specify an IP address range (for example, 203.0.113.5/32)
```
Comment (acarranoqovery) [inline openapi.yaml:13035]: add a more complex example like ['203.0.113.5/32','203.0.100/32']

File: openapi.yaml (hunk 1 above)
Comment (pggb25) [reply openapi.yaml]: done

File: openapi.yaml (hunk 2 above)
Comment (pggb25) [reply openapi.yaml]: done

**Review by acarranoqovery — APPROVED** [review]

**Pull request merged**